packages/compare-slider-block/src/CompareSliderBlock.tsx

```diff
@@ -0,0 +1,397 @@
+            (assetWithSmallerAspectRatio.height * currentSliderWidth) / assetWithSmallerAspectRatio.width
+        );
+    };
+
+    const getImageHeight = (): number | undefined => {
+        if (hasCustomHeight) {
+            return parseInt(customHeight);
+        }
+
+        if (height !== Height.Auto) {
+            return heightMap[height];
+        }
+
+        return calculateAutoImageHeight();
+    };
+
+    const getBorderStyle = () => {
+        return hasBorder ? { borderWidth, borderColor: toRgbaString(borderColor), borderStyle } : {};
+    };
+
+    const getBorderRadius = () => {
+        if (hasRadius) {
+            return radiusValue;
+        }
+
+        return radiusStyleMap[radiusChoice];
+    };
+
+    const renderSliderItem = (slot: SliderImageSlot) => {
+        return (
+            <div>
+                <div style={{ background: hasBackground ? toRgbaString(backgroundColor) : undefined }}>
+                    <img
+                        src={slot === SliderImageSlot.First ? firstAssetPreviewUrl : secondAssetPreviewUrl}
+                        alt={slot === SliderImageSlot.First ? firstAssetTitle : secondAssetTitle}
+                        className="tw-w-full tw-object-cover"
+                        style={{ height: getImageHeight() }}
+                    />
+                </div>
+
+                {renderStrikethrough(slot)}
+                {!isEditing && renderLabel(slot)}
+            </div>
+        );
+    };
+
+    const renderStrikethrough = (slot: SliderImageSlot) => {
+        let top = 0;
+        let left = 0;
+        let height = alignment === Alignment.Horizontal ? 100 : currentSliderPosition;
+        let width = alignment === Alignment.Horizontal ? currentSliderPosition : 100;
+
+        if (slot === SliderImageSlot.Second) {
+            top = alignment === Alignment.Horizontal ? 0 : currentSliderPosition;
+            left = alignment === Alignment.Horizontal ? currentSliderPosition : 0;
+            height = alignment === Alignment.Horizontal ? 100 : 100 - currentSliderPosition;
+            width = alignment === Alignment.Horizontal ? 100 - currentSliderPosition : 100;
+            if (!secondAssetHasStrikethrough) {
+                return null;
+            }
+        }
+        if (slot === SliderImageSlot.First && !firstAssetHasStrikethrough) {
+            return null;
+        }
+        return (
+            <StrikethroughWrapper top={top} height={height} width={width} left={left} borderRadius={getBorderRadius()}>
+                <Strikethrough
+                    color={
+                        strikethroughColorSource === InheritSettings.OVERRIDE
+                            ? toRgbaString(customStrikeThroughColor)
+                            : designTokens.callout?.warning ?? toRgbaString(DEFAULT_STRIKETHROUGH_COLOR)
+                    }
+                />
+            </StrikethroughWrapper>
+        );
+    };
+
+    const renderLabel = (slot: SliderImageSlot) => {
```

Comment:
Also here consider to put everything directly into the LabelWrapper

Reply:
Same here, implemented 👍